### ISSUE TITLE
Unable to build the repo on mac

### ISSUE THREAD

**Issue by krishnaw14** [aerospaceresearch/orbitdeterminator#110]
On running `$ python setup.py install`: I am getting the error as: 
`Searching for pykep
Reading https://pypi.python.org/simple/pykep/
No local packages or working download links found for pykep
error: Could not find suitable distribution for Requirement.parse('pykep')`

I was unable to find a work around this. This might be something trivial but I am new to this so I apologise in advance.

**Comment by PerezHz** [comment]
I'm having a similar issue and trying to figure out how to fix this, what version of macOS are you using @krishnaw14?

**Comment by krishnaw14** [comment]
@PerezHz I am using MacOS High Sierra- version 10.13.2

**Comment by PerezHz** [comment]
I'm also a new user and am also on macOS High Sierra (10.13.3); the way I worked around this issue was the following:

- `brew install` python, boost and boost-python3
- Install pykep from source, following the guidelines in the pykep docs, which you can find [here](https://esa.github.io/pykep/installation.html)
- Then, running python from the `orbitdeterminator/orbitdeterminator` dir I was able to do:

```python
$ python
Python 3.6.4 (default, Mar  9 2018, 23:15:03) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import main
>>> run = main.process("orbit.csv", 20)
/usr/local/lib/python3.6/site-packages/scipy-1.0.0-py3.6-macosx-10.13-x86_64.egg/scipy/linalg/basic.py:1226: RuntimeWarning: internal gelsd driver lwork query error, required iwork dimension not returned. This is likely the result of LAPACK bug 0038, fixed in LAPACK 3.2.2 (released July 21, 2010). Falling back to 'gelss' driver.
  warnings.warn(mesg, RuntimeWarning)
Displaying the sum of the residuals for each axis
[ 361.73846007778263   35.16624820681119 -414.35666149725444]
 
Displaying the mean of the residuals for each axis
[ 0.0452173075097228  0.0043957810258514 -0.0517945826871568]
 
Displaying the final keplerian elements, first row : Lamberts, second row : Interpolation
[[6.7921084958355223e+03 6.8045188830705629e+03]
 [1.4795216724147046e-03 8.8847298704285680e-04]
 [5.1670954834894410e+01 5.1675000118622904e+01]
 [2.1508751005933266e+02 1.9537230526813295e+02]
 [1.1250005890032938e+02 1.1251077719782269e+02]
 [1.6833468551374364e+02 1.6833468551374364e+02]]
```

So, things seem to be working even if `python setup.py install` fails. I haven't been able to figure out if this issue is related to the fact that pykep wheels don't seem to be available for macOS, or either to some version issue with Homebrew's Boost library, since `cmake` seems to complain about the Boost version while building `pykep` from source. Either way, `orbitdeterminator` docs state that the preferred way to run things is directly from the package source dir, without installing it. I'll keep trying to figure out what's going on. Hope this helps!

**Comment by Alexandros23Kazantzidis** [comment]
Hello guys, unfortunately, we have not encountered this problem yet due to the fact we mainly use ubuntu and windows. I don't know why this error with the pykep library occurs but I think the last solution works. The output you get is the standard output of the main.py script.
